core/src/main/java/ar/com/wolox/wolmo/core/util/SharedPreferencesManager.java

```diff
@@ -28,66 +28,66 @@
 import javax.inject.Inject;
 
 /**
- * Storage utils to query and store values in {@link SharedPreferences}.
+ * Utility class to query and store values in {@link SharedPreferences}.
  */
 @ApplicationScope
-public class StorageUtils {
+public class SharedPreferencesManager {
```

Comment:
Mmm, instead of making all methods `final` (which makes sense in this case), why don't we `final`ize the class instead?